### DISCUSSION TITLE
우당탕탕 JPA | Devdalus Wiki

### DISCUSSION THREAD

**Discussion by giscus** [DevdaIus/DevdaIus.github.io#2] [General]
# 우당탕탕 JPA | Devdalus Wiki

Tech Archive for project Devdalus

https://devdaius.github.io/docs/database/jpa/jpa_for_dummies.html

**Comment by punkryn** [comment]
잘 보이네요 👍
![image](https://gist.github.com/assets/22855979/8380477d-f040-4c17-b4e6-2aa50e020ba7)
